odoo/addons/base/models/ir_actions.py

```diff
@@ -671,7 +671,7 @@ def _compute_crud_relations(self):
                 elif action.state == 'object_write':
                     if action.update_path:
                         # we need to traverse relations to find the target model and field
-                        model, field, _ = action._traverse_path()
+                        model, field = action._traverse_path("update_path")
```

Comment:
Shouldn't we have a warning msg in base that could check for non existing paths ? Not just specific to mail but could maybe be useful in base ? Maybe it is already supported, did not try to play with server actions recently :)

Reply:
Just that there is a warning in 'mail', but do we have warnings in base module or simply errors ? Looking at code, "invalid" path in mail triggers a warning, but "non existing" paths raise an error, is that the behavior ?